code/math/vecmat.h

```diff
@@ -477,7 +481,14 @@ void vm_vec_dist_squared_to_line(const vec3d *p, const vec3d *l0, const vec3d *l
 //SUSHI: 2D vector "box" scaling
 void vm_vec_boxscale(vec2d *vec, float scale);
 
-bool vm_inverse_matrix4(const matrix4 *m, matrix4 *invOut);
+// adds one matrix to another.
+// dest can equal source
+// dest += src
+void vm_matrix_add2(matrix* dest, const matrix* src);
+
+bool vm_inverse_matrix(matrix* dest, const matrix* m);
+
+bool vm_inverse_matrix4(const matrix4* m, matrix4* invOut);
```

Comment:
If you are going to make a separate vecmat PR (which I recommend), then it would be good to swap the arguments of `vm_inverse_matrix4` and fix the null matrix bug I mentioned.